src/styles/app.css

```
@@ -53,7 +53,7 @@
   --color-sidebar-ring: var(--sidebar-ring);
   --font-sans: Outfit, sans-serif;
   --font-mono: monospace;
-  --font-serif: ui-serif, Georgia, Cambria, "Times New Roman", Times, serif;
+  --font-serif: ui-serif, Georgia, Cambria, 'Times New Roman', Times, serif;
```

Stylelint (17.6.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)